### PR TITLE
Fix DDP_popen slot issues

### PR DESCRIPTION
**Pull request by futujaos**

### CODE REVIEW

File: lzy/fs/src/main/java/ai/lzy/fs/slots/InFileSlot.java [hunk 1]
```diff
@@ -153,7 +153,7 @@ public void close() throws IOException {
                 LOG.info("Closing file {} for slot {}", storage.toString(), name());
                 channel.close();
                 trackers().forEach(ContentsTracker::onClose);
-                state(State.SUSPENDED);
+                //state(State.SUSPENDED);
```
Comment (imakunin) [inline lzy/fs/src/main/java/ai/lzy/fs/slots/InFileSlot.java:156]: add test, plz
https://paste.yandex-team.ru/fb7fc8b5-6da4-4b05-a087-27b21ec4cf6e/text

File: lzy/fs/src/test/java/ai/lzy/fs/LzyFSTest.java [hunk 2]
```diff
@@ -62,6 +62,51 @@ public void testFuseConfiguration() {
         System.out.println("Ok");
     }
 
+    @Test
+    public void testMultipleRead() throws URISyntaxException, IOException {
```
Comment (imakunin) [inline lzy/fs/src/test/java/ai/lzy/fs/LzyFSTest.java:66]: I already did it :)

File: pylzy/lzy/api/v1/startup.py [hunk 3]
```diff
@@ -89,6 +90,10 @@ def process_execution(
     logger: Logger,
     lazy_arguments: bool
 ):
+    pid, ppid = os.getpid(), os.getppid()
+    main_pid = os.getenv(MAIN_PID_ENV_VAR, None)
+    logger.debug(f"Starting process with pid {pid} (ppid {ppid}, main pid {main_pid})")
```
Comment (imakunin) [inline pylzy/lzy/api/v1/startup.py:95]: @vhaldemar is categorically against the use of f-formatting in logs

File: pylzy/lzy/api/v1/startup.py [hunk 4]
```diff
@@ -118,6 +123,14 @@ def process_execution(
         raise
     logger.info(f"Execution completed in {time.time() - start} sec")
 
+    write_results = main_pid is None or int(main_pid) == pid
+    if not write_results:
```
Comment (imakunin) [inline pylzy/lzy/api/v1/startup.py:127]: we should test exceptions here
all exceptions thrown by the `op` are written to a special output slot (exceptions slot)

Comment (futujaos) [reply pylzy/lzy/api/v1/startup.py:127]: you mean we should write exceptions from subprocesses?
but there is only a single exception slot (same problem with multiple writes) and we are probably interested only in exceptions from "main" process

**Comment by futujaos** [comment]
Change default value of `allowMultipleRead` to `true`, log multiple reads.

**Review by imakunin — APPROVED** [review]

**Comment by imakunin** [comment]
LGTM, the only thing is to change the type of `readCount` from `int` to `AtomicInteger`

**Review by imakunin — APPROVED** [review]

**Pull request merged**